privaterelay/templates/home.html

```diff
@@ -1,15 +1,18 @@
 {% extends "base.html" %}
 
+{% load ftl %}
 {% load socialaccount %}
 
 {% block content %}
 
 {% include "includes/messages.html" %}
 
+{% withftl bundle='privaterelay.ftl_bundles.main' %}
```

Comment:
I used the [`withftl`](https://django-ftl.readthedocs.io/en/latest/usage.html#withftl) method, rather than including the file via [`ftlconf`](https://django-ftl.readthedocs.io/en/latest/usage.html#ftlconf). This can be revised as needed.